app/src/main/java/org/traccar/client/MainFragment.java

```diff
@@ -191,6 +193,18 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         super.onCreateOptionsMenu(menu, inflater);
     }
 
+    // Aina 21-Jan-2019, Emergency alert on broadcast
+    @Override
+    public void onPrepareOptionsMenu(Menu menu) {
```

Comment:
Why is it disabled if status is ON?

Reply:
Because it is common behaviour of all preferences and if you change broadcast value in service run-time, changes won't apply.